src/data/polynomial/ring_division.lean

```diff
@@ -671,25 +671,15 @@ lemma bUnion_roots_finite {R S : Type*} [semiring R] [comm_ring S] [is_domain S]
   (m : R →+* S) (d : ℕ) {U : set R} (h : U.finite) :
   (⋃ (f : R[X]) (hf : f.nat_degree ≤ d ∧ ∀ i, (f.coeff i) ∈ U),
     ((f.map m).roots.to_finset : set S)).finite :=
```

Comment:
`root_set f S` is more succinct and defeq to `((f.map m).roots.to_finset : set S)`, but would require we replace `m : R →+* S` by `algebra R S`.

Reply:
Although `R →+* S` fits more naturally with the use we have for it for `number_field`, since it is a result in `polynomial/ring_division`, it does not have to be tailored for number fields and you should use whatever you think makes the lemma better.

Reply:
Okay, let's keep this as is.